test/unit/storage/getBlobServiceClient.test.js

```diff
@@ -0,0 +1,36 @@
+import { BlobServiceClient } from '@azure/storage-blob'
+import { getBlobServiceClient } from '../../../app/storage/getBlobServiceClient'
+
+const mockFromConnectionString = jest.fn()
+jest.mock('@azure/storage-blob')
+BlobServiceClient.fromConnectionString = mockFromConnectionString
+
+const constructorSpy = jest.spyOn(
+  require('@azure/storage-blob'),
```

Comment:
This still good to go with it using a require?

Reply:
Yeah its needed unfortunately - its the only way that we can check if BlobServiceClient is called using .fromConnectionString or via the constructor